src/app/shared/base-components/upload-excel/upload-excel.component.scss

```diff
@@ -0,0 +1,221 @@
+@import 'src/app/shared/styles/buttons.scss';
+
+$duplicatedValue: #fd0707b3;
+$emptyField: #fd07071a;
+$wrongLength: #0d138980;
+$wrongLanguage: #22b01280;
+$wrongFormat: #ffff0080;
+$lengthWarningMessage: #ff0000;
+
+.import-buttons {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  .submit-btn {
+    margin-left: 1px;
+  }
+  .file-name {
+    margin-left: 1rem;
+  }
+  .btn {
+    text-transform: uppercase;
+    height: 40px;
+    margin-bottom: 10px;
+  }
+  .mat-raised-button {
+    border-radius: 20px;
+    height: 40px;
+    margin-bottom: 10px;
+    &:hover {
+      color: white;
+      background-color: #3849f9;
+    }
+  }
+}
+.file-actions {
+  display: flex;
+}
+.errors-menu {
+  display: flex;
+  width: fit-content;
+  .length-warning {
+    max-width: 200px;
+    line-height: 15px;
+    mat-card-subtitle {
+      color: $lengthWarningMessage !important;
+      animation: pulse 3s infinite;
+    }
+  }
+  mat-slide-toggle {
+    padding-top: 5px;
+  }
+  .incorrect-row {
+    color:red;
+  }
+  .incorrect-row-number {
+    color: red;
+  }
+}
+.errors-types {
+  margin: 20px 20px;
+  ul {
+    padding: 0;
+    list-style-type: none;
+    li {
+      display: flex;
+      flex-direction: row;
+      justify-content: start;
```

Comment:
It is better to use flex-start, as the autoprefixer may not work correctly in some browsers